### PR TITLE
Env based referral link preview

### PR DESCRIPTION
**Pull request by jvonasek**

### CODE REVIEW

File: src/sections/referrals/components/CodePreview/CodePreview.tsx [hunk 1]
```diff
@@ -20,24 +20,35 @@ type Props = {
   disabled?: boolean
 }
 
+const IS_PROD = import.meta.env.PROD
```
Comment (vkulinich-cl) [inline src/sections/referrals/components/CodePreview/CodePreview.tsx:23]: is that a valid variable?

Comment (vkulinich-cl) [reply src/sections/referrals/components/CodePreview/CodePreview.tsx]: there is a VITE_ENV=production variable. It's better I think

File: src/sections/referrals/components/CodePreview/CodePreview.tsx [hunk 2]
```diff
@@ -20,24 +20,35 @@ type Props = {
   disabled?: boolean
 }
 
+const IS_PROD = import.meta.env.VITE_DOMAIN_URL === "https://app.hydradx.io"
+const PROD_HOST = "hydradx.io"
+const REF_PARAM_NAME = "referral"
+
+function getShareUrl(code: string, origin?: string) {
```
Comment (vkulinich-cl) [inline src/sections/referrals/components/CodePreview/CodePreview.tsx:27]: Could you move that code to utils file

File: src/sections/referrals/components/CodePreview/CodePreview.tsx (hunk 1 above)
Comment (jvonasek) [reply src/sections/referrals/components/CodePreview/CodePreview.tsx]: true, didnt see it, fixed

File: src/sections/referrals/components/CodePreview/CodePreview.tsx (hunk 2 above)
Comment (jvonasek) [reply src/sections/referrals/components/CodePreview/CodePreview.tsx]: done

**Pull request merged**